vitrivr-engine-server/src/main/kotlin/org/vitrivr/engine/server/Main.kt

```diff
@@ -31,7 +31,8 @@ fun main(args: Array<String>) {
 
     /* Setup schema manager. */
     val manager = SchemaManager()
-    for (schema in config.schemas) {
+    for ((name, schema) in config.schemas) {
+        schema.name = name
```

Comment:
This logic does not really belong here. It makes sense that the schema knows its own name, but that indirection should be resolved in the config object itself.

Reply:
Thank you for your review! I understand your point. Would it be preferable to assign the name in the ServerConfig (during read) or in an extra step with a dedicated method in the SchemaConfig?

Reply:
It would make sense to have this inside an `init` block inside the schema config; then, it would also be called once this is deserialized.